### DISCUSSION TITLE
Gruppe-4 Discussions

### DISCUSSION THREAD

**Discussion by nezrinstyles** [nezrinstyles/Gruppe-4#5] [Announcements]
##  Welcome!
  Hier besprechen wir alles, was mit unseren Aufgaben zu tun hat.

**Comment by nezrinstyles** [comment]
Hey Mädels, da Sissi und ich mit Aufgabe 1 und 2 fertig sind, ist es Zeit für uns, mit Aufgabe 3 zu beginnen, die wir gemeinsam erledigen müssen

**Comment by xi-wa** [comment]
Sollten wir dann erstmal besprechen, welche Funktionen wir auf jeden Fall drinnen haben wollen? Das ist etwa der letzte Punkt von Aufgabe 3
Wir können das aber auch anders machen, wenn jemand eine andere Idee hat

**Reply by nezrinstyles** [reply]
Ich denke wir können auch Freiwillig einen Funktion zur Erstellung eines Histogramms, das die Verteilung der Altersgruppen von Studenten(-inen) in einem bestimmten Studienfach darstellt

**Reply by nezrinstyles** [reply]
Oder zum Beispiel Funktion zum Balkendiagramms, das einen Anteil der Studenten mit einem hohen Interesse an Mathe in jedem Studienfach darstellt wird

**Reply by YuxinZhuang** [reply]
ich habe eine Idee. für (a) sehen wir nur 2-6 Spalten,dann können wir eine Funktion schreiben, ob die Variablen zu 2-6 gehören,wenn ja ,können wir deskriptive Statistik machen, wenn nein, return(Variable muss in 2-6).

**Reply by xi-wa** [reply]
Ich bin mir nicht sicher, ob ich das verstanden habe
Meinst du überprüfen ob es eine metrische oder kategoriale Variable ist?

**Reply by YuxinZhuang** [reply]
ich meine, wir können eine Funktion schreiben, ob die Variablen von Funktion a-f  gültig ist.

**Reply by xi-wa** [reply]
Ich glaube wir meinen in etwa dasselbe
Aber das ist eine gute Idee
Schreibst du dann ebenfalls die Funktionen?

**Reply by YuxinZhuang** [reply]
ja,kein Problem,ich kann später hochladen

**Comment by nezrinstyles** [comment]
Nun, wir müssen auch die Punkte a-f verteilen, die im Code enthalten sein müssen. Gibt es Vorlieben? Wir sind nur 5, aber 6 Punkte und jemand muss noch einen nehmen.

**Reply by xi-wa** [reply]
Mir ist es egal was ich mache
Ich kann a+b machen, die sind sich relativ ähnlich
Ist es ok für euch oder möchte es jemand lieber machen?

**Reply by nezrinstyles** [reply]
Mir ist auch egal, wenn du (a)+(b) nimmst dann nehme ich (c), falls sonst keiner genau diesen Punkt will

**Reply by YuxinZhuang** [reply]
kein Problem,,ich kann (d) machen.und ich habe auch (a)(b) geschrieben.

**Reply by aYajie** [reply]
OK, dann kann ich (e) machen.

**Reply by xi-wa** [reply]
@YuxinZhuang Willst du dann (a)+(b) machen, wenn du das schon hast?
Ich kann auch d machen, ist kein Problem

**Reply by YuxinZhuang** [reply]
Aber ich habe schon die Lösung von A3(e) hochgeladen, und ich werde sofort die Funktionen-Skript 2 auch hochladen.

**Reply by nezrinstyles** [reply]
Warum machst du (e) wenn Yajie es machen wollte und du wolltest (d) machen?

**Reply by YuxinZhuang** [reply]
Entschuldigung, ich habe falsch gesagt, ich mache (d) :)

**Reply by YuxinZhuang** [reply]
habt ihr alle die sellsbt Lösung von A3 schreiben?und dann kann ich die Hilfsfunktion hochladen.

**Reply by nezrinstyles** [reply]
Ich hab beim Pull Request Aufgabe 3 erstellt, ihr könnt einfach die Datei bearbeiten und ihre Teil hinzufügen, dann mache ich alles Merge damit es nur 1 Datei bleibt

**Comment by xi-wa** [comment]
Ich hab übrigens die Codierung von Mathe LK geändert, TRUE = ja und FALSE = nein
Aber ihr solltet euch das eigentlich nicht anschauen

**Reply by nezrinstyles** [reply]
Hab gesehen, danke Sissi:)

**Reply by aYajie** [reply]
Ja,alles klar

**Comment by YuxinZhuang** [comment]
hallo, jetzt wir drei, die nicht an A1 teilnehmen, machen wir die Aufgabe 4!

**Reply by HongmingQi** [reply]
> hallo, jetzt wir drei, die nicht an A1 teilnehmen, machen wir die Aufgabe 4!

Ich komme .

**Reply by aYajie** [reply]
Los geht's, lasst uns die Diskussion beginnen :)

**Reply by HongmingQi** [reply]
Es gibt ein Problem, vielleicht müssen wir die Adresse der Datei hier ändern. z.b (source("Hilfsfunktion.R")
source("WA github.R")
)

**Reply by YuxinZhuang** [reply]
ne, ich werde noch zwei Datei hochladen

**Reply by HongmingQi** [reply]
Perfekt 🤩

**Comment by xi-wa** [comment]
Zu der Funktion (e):
Soweit ich das verstanden habe, gibt diese Funktion die Werte in Quantile wieder aus. Das Quantil hier war ja low, medium, high.
Also sind beispielsweise jene die bei "Interesse an Mathe" nur 1 angegeben haben "low"
War das verständlich erklärt?

**Reply by HongmingQi** [reply]
> Zu der Funktion (e):
> Soweit ich das verstanden habe, gibt diese Funktion die Werte in Quantile wieder aus. Das Quantil hier war ja low, medium, high.
> Also sind beispielsweise jene die bei "Interesse an Mathe" nur 1 angegeben haben "low"
> War das verständlich erklärt?

Das war's also, vielen Dank.
